app/code/community/Bolt/Boltpay/sql/bolt_boltpay_setup/upgrade-1.4.1-1.4.1.1.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
This file seems overly complicated; can we instead handle empty value when reading config? eg., `@$this->getExtraConfig('datadogKey') || $defaultApiKey`